tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/auth/OAuthConfigTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.ballerinalang.test.auth;
+
+import org.ballerinalang.test.context.BMainInstance;
+import org.ballerinalang.test.context.LogLeecher;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+/**
+ * Testing OAuth Client Authentication Configs.
+ */
+@Test(groups = "auth-test")
+public class OAuthConfigTest extends AuthBaseTest {
+
+    private BMainInstance ballerinaClient;
+    private static final String BAL_FILE = new File("src" + File.separator + "test" + File.separator + "resources"
```

Comment:
I think we can use `Paths` here to simplify the code.

Eg- `Paths.get("src", "test", "resources", ... "oauth-client.bal").toFile().getAbsolutePath()`